src/dreyfus_index.erl

```diff
@@ -369,3 +383,23 @@ group2_int(Pid, QueryArgs0) ->
 
 info_int(Pid) ->
     clouseau_rpc:info(Pid).
+
+verify_index_exists(Options) ->
+    DbName = dreyfus_util:get_value_from_options(<<"db_name">>, Options),
```

Comment:
These keys are different than what the change to couch_mrview added. We should settle on the naming scheme (even though this would work fine, consistency is always better).